tensorflow/stream_executor/rocm/rocm_fft.cc

```diff
@@ -71,7 +71,7 @@ namespace wrap {
   __macro(hipfftGetSizeMany)         \
   __macro(hipfftMakePlanMany)        \
 
-  // XXX disable before rocFFT uses proper HIP complex types
+  // ROCM TODO disable before rocFFT uses proper HIP complex types
```

Comment:
Do we have an ETA when TF requirements will change to ROCm 1.8.2 and we can enable this?